frontend/src/components/v2/Button/ContextMenuButton.vue

```diff
@@ -77,7 +77,9 @@ const currentActionKey = ref(getDefaultActionKey());
 const currentAction = computed(() => {
   const key = currentActionKey.value;
   if (!key) return undefined;
-  return props.actionList.find((action) => action.key === key);
+  return (
+    props.actionList.find((action) => action.key === key) ?? props.actionList[0]
```

Comment:
using `lodash.head(props.actionList)` instead of `props.actionList[0]`

Reply:
updated